docs/my-classic-app-with-lazy-loaded-translations/ember-cli-build.js

```diff
@@ -10,6 +10,7 @@ module.exports = function (defaults) {
     },
 
     fingerprint: {
+      enabled: true,
```

Comment:
To document that, for apps with classic build, `ember-intl` works when fingerprinting is enabled.